js/main.js

```diff
@@ -1,2 +1,125 @@
 $(function () {
+
+// Rock Paper Scissor routine input user choice, generate random computer response
+// find out who won, update score and then present result in table on screen
+// There seems to be a bias in the program toward the user -- cause not clear
+// First define globale variables that are used to display results or simplify flow:
+  let userWins=0
+  let machineWins=0
+  let machineInput=NaN
+  let userInput=NaN
+
+  // The evenhandler is where we are waiting for input
+  // Very much in contrast to "normal linear programming"
+  // Everything emenates from here
+
+  // If the user clicks "rock":
+  $('#rock').click(() => {
+  let userInput = $('#rock').text()
```

Comment:
@AtleLohrmann you can also grab the text from `$(this)` because `this` would refer to the object clicked.
Also, because we know which button was clicked, you can use an inline string with the respective button's value.

check out [this](https://github.com/jsr20190826/rock-paper-scissors-solution/blob/1583be48146e2521b25308e017c52fbfcf0d9d89/js/main.js#L9) approach